webcompat/views.py

```diff
@@ -73,10 +73,15 @@ def format_date(datestring):
 @app.route('/login')
 def login():
     if session.get('user_id', None) is None:
-        # manually set the referer so we know where to come back to
-        # when we return from GitHub
-        set_referer(request)
-        return github.authorize('public_repo')
+        if app.config['TESTING']:
+            session['username'] = 'testuser'
+            session['avatar_url'] = '/img/avatar.png?'
```

Comment:
Also, can we put this image in the `/tests/fixtures` folder? It feels a bit odd to expose this image to the web. 

If you use the `test-files` route (in views.py) `http://localhost:5000/test-files/fixtures/avatar.png` should work.